### PR TITLE
[14.0][MIG] sale_product_pack: migrate to 14.0

### PR DESCRIPTION
**Pull request by dreispt**
Superseeds #34

### CODE REVIEW

**Comment by Steff-mwl** [comment]
Hi,
Is it planned to integrate the module soon ?

**Comment by pedrobaeza** [comment]
@Steff-mwl All starts doing a review for assuring the correct functionality: https://odoo-community.org/page/review

@sbidoul it seems the bot is again stalled.

**Comment by sbidoul** [comment]
GitHub has destroyed the bot token again... please retry.

**Review by pedrobaeza — APPROVED** [review]

File: sale_product_pack/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 NaN (http://www.nan-tic.com) - Àngel Àlvarez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Sale Product Pack",
+    "version": "14.0.1.0.2",
```
Comment (ernestotejeda) [inline sale_product_pack/__manifest__.py]:
```suggestion
    "version": "14.0.1.0.0",
```

**Comment by alielnagar** [comment]
hello need this app please

File: sale_product_pack/__manifest__.py (hunk 1 above)
Comment (rvalyi) [reply sale_product_pack/__manifest__.py]: @dreispt could you just change the version as requested so we can merge it?

Comment (dreispt) [reply sale_product_pack/__manifest__.py]: Done!

**Pull request merged**